flake.nix

```diff
@@ -465,6 +465,10 @@
 
             src = self;
 
+            configureFlags = [
+              "CXXFLAGS=-I${lib.getDev pkgs.rapidcheck}/extras/gtest/include"
```

Comment:
Why do we pass this as a configure flag? Why not `NIX_CFLAGS_COMPILE`?

Reply:
@edolstra It could work either way, but I think this is a bit nicer because it documents how one would do things without a cc-wrapper on other platforms.

I opened https://github.com/emil-e/rapidcheck/issues/302 in hope that rapidcheck would be OK with having a pkg-config file, and then all this can just go away.